sklearn/multiclass.py

```diff
@@ -514,6 +514,21 @@ class OneVsOneClassifier(MetaEstimatorMixin, ClassifierMixin, BaseEstimator):
     pairwise_indices_ : list, length = ``len(estimators_)``, or ``None``
         Indices of samples used when training the estimators.
         ``None`` when ``estimator`` does not have ``_pairwise`` attribute.
+
+    Examples
+    --------
+    >>> from sklearn import datasets
+    >>> from sklearn.multiclass import OneVsOneClassifier
+    >>> from sklearn.svm import LinearSVC
+    >>> X, y = datasets.load_iris(return_X_y=True)
```

Comment:
```suggestion
    >>> X, y = load_iris(return_X_y=True)
```

Reply:
Let's also split the data into training-testing using `train_test_split` utility.